content/06.discussion.md

```diff
@@ -54,6 +54,34 @@ In assessing the upper bound on the predictive performance of a deep learning mo
+
+#### Test-time dropout
+
+Gal and Ghahramani showed that a stack of fully connected layers with dropout between the layers is equivalent to approximate inference in a Gaussian process model [@tag:Gal2015_dropout]. The authors interpret dropout as a variational inference method and apply their method to convolutional neural networks. This is simple to implement and preserves the possibility of obtaining cheap samples from the approximate posterior distribution. Operationally, obtaining model uncertainty for a given case becomes as straightforward as leaving dropout turned on and predicting multiple times. The spread of the different predictions is a reasonable proxy for model uncertainty. This technique has been successfully applied in an automated system for detecting diabetic retinopathy [@tag:retinopathy_uncertainty], where uncertainty-informed referrals improved diagnostic performance and allowed the model to meet the NHS recommended levels of sensitivity and specificity. The authors also found that entropy performs comparably to the spread obtained via test-time dropout for identifying uncertain cases, and therefore it can be used instead for automated referrals.
+
+#### Bayesian hypernetworks
+
+Another framework for approximate Bayesian inference in deep learning is utilizing Bayesian hypernetworks [@tag:bayesian_hypernets]. A hypernetwork is a network that learns to map its input to the parameters of another network, the primary network. Both the hypernetwork and the primary network are trained together as a single model. Krueger et al introduced Bayesian hypernetworks and used an invertible generative network to map isotropic Gaussian noise to parameters of the primary network. Using an invertible hypernetwork allowed computationally cheap sampling and tractable estimation of the posterior. Bayesian hypernetworks are better at dealing with multimodal posterior distributions than other Bayesian deep learning approaches.
```

Comment:
et al. again

Reply:
Actually, everywhere. Remeber "et al. " is short for "et alia", so it's an abbreviation and needs the punctuation.